hera_librarian/client.py

```diff
@@ -733,3 +733,51 @@ def add_file_row(
                 raise LibrarianError(f"Unknown error. {e}")
 
         return response
+
+    def verify_file_row(
+        self,
+        name: str,
+    ):
+        """
+        Verify a file row against an existing file on the store.
+        This can confirm the integrity and existence of a file as recorded in the database.
+
+        Parameters
+        ----------
+        name : str
+            The unique filename of the file to verify.
+
+        Returns
+        -------
+        dict
+            A dictionary indicating whether the file is verified.
+
+        Raises
+        ------
+        LibrarianError
+            If the verification fails or the file/store does not exist.
+        """
+
+        try:
+            response = self.post(
+                endpoint="admin/verify_file",
+                json={
+                    "name": name,
+                },
+            )
+
+            if response.status_code != 200:
+                raise LibrarianError(
+                    "Failed to verify the file due to an unexpected error."
+                )
+            return response.json()
+
+        except LibrarianHTTPError as e:
+            if e.status_code == 404:
+                raise LibrarianError("File or store not found for verification.")
+            elif e.status_code == 400:
+                raise LibrarianError(
+                    "File verification failed due to mismatched properties."
+                )
+            else:
+                raise LibrarianError(f"Unknown error during file verification. {e}")
```

Comment:
As I said before, please make sure you serialize/deserialize everything to pydantic models. In fact, I don't think `json` is even an accepted parameter in our post function! Please can you also expand your test coverage to make sure that these are checked?